### ISSUE TITLE
not working 

### ISSUE THREAD

**Issue by extreme-b0t** [adtitas/ngrok-rdp#17]
RDP not working, and it only stay for mins not 5-6 hours.

**Comment by adtitas** [comment]
Did you follow the instructions step by step?

**Comment by extreme-b0t** [comment]
Yes. Ngrok not opening, and RDP only stay for 30-40mins. Not 5-6 hours

**Comment by adtitas** [comment]
It's now 1 hour.
I'm working on it to make again 5-6 hours.
